### PR TITLE
Feat/admin show total invoice revenue 2

### PR DESCRIPTION
**Pull request by cballrun**
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please describe):
## Problem/feature
_What problem are you trying to solve?_
Display of total invoice revenue in dollars on Admin Invoice show page
## Solution
_How did you solve the problem?_
Invoice model instance methods total_invoice_revenue and total_invoice_revenue dollars/associated tests
Add line for total revenue to Admin Invoice Show page in invoice_info section/associated test
## Other changes (e.g. bug fixes, UI tweaks, small refactors)
FE updates to admin invoice show page to match wireframe more closely
Fix test in invoice model spec to not fail
Candace's requested updates: number_with_precision
## Checklist
- [x] Code compiles correctly
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: app/models/invoice.rb [hunk 1]
```diff
@@ -30,4 +30,14 @@ def self.incomplete_invoices_sorted
...
   end
+
+  def total_invoice_revenue
+    items
+    .joins(:invoice_items)
+    .sum("invoice_items.unit_price * invoice_items.quantity")
+  end
+
+  def total_invoice_revenue_dollars 
+    total_invoice_revenue.to_f / 100
```
Comment (mullinsand) [inline app/models/invoice.rb:41]: I think this might need to be view logic

**Review by mullinsand — COMMENTED** [review]
two things: 1. I think the total_invoice_revenue_dollars should probably be a view action and not a model but I'm open to feedback.
2. total_invoice_revenue: noting for refactor time, add test for the no invoice_items on an invoice edge case and another for multiple invoice_items on a single invoice.
